pkg/sql/schemachanger/sctest/framework.go

```diff
@@ -48,12 +48,13 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+//go:generate stringer -type=stageExecType
```

Comment:
I am not familiar with this go-gen, what's this for?

Reply:
It code gens the `stageexectype_string.go` file.